Calendar/BookingHelpers/BookingStaffHelpers.ps1

```diff
@@ -0,0 +1,54 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Get-MBPermissions {
+    param($Identity)
+    # Get the Mailbox Permissions
+    $MBPermissions = Get-MailboxPermission -Identity $Identity -ErrorAction SilentlyContinue
+    return $MBPermissions
+}
+
+function Get-MBRecipientPermissions {
+    param($Identity)
+    # Get the Mailbox Recipient Permissions
+    $MBRecipientPermissions = Get-RecipientPermission -Identity $Identity -ErrorAction SilentlyContinue
+    return $MBRecipientPermissions
+}
+
+function CheckMyBaseOptionsRBACRole($Identity) {
+    $RBACRole = Get-ManagementRoleAssignment -RoleAssignee $Identity -Role "MyBaseOptions" | Select-Object -Property RoleAssignee, Role
+    return $RBACRole
+}
+
+function CheckPersistedCapabilities($Identity) {
+    $PC = Get-Mailbox -Identity $Identity | Select-Object -ExpandProperty PersistedCapabilities
```

Comment:
Shouldn't need to do this, could do this from the original `Get-Mailbox` which is set to a variable `$Script:BMB`

```
$PC = $script:BMB | Select-Object -ExpandProperty PersistedCapabilities
```

Reply:
This is not for the Bookings Mailbox, this is called for the Staff Mailbox Address, so this information is not in Cache.
I'm adding a comment for this method to document that.